plugins/metrics/metrics.go

```diff
@@ -45,36 +44,29 @@ type config struct {
 type metricsInstaller struct{}
 
 func newMetricsMiddleware(namespace string) definition.Middleware {
+	constLabel := prometheus.Labels{"component": namespace}
 	requestCounter := prometheus.NewCounterVec(
 		prometheus.CounterOpts{
-			Namespace: namespace,
-			Name:      "request_count",
-			Help:      "Counter of server requests broken out for each verb, API resource, client, and HTTP response contentType and code.",
+			Namespace:   namespace,
+			Name:        "request_total",
+			Help:        "Counter of server requests broken out for each verb, API resource, and HTTP response code.",
+			ConstLabels: constLabel,
 		},
-		[]string{"method", "path", "client", "contentType", "code"},
```

Comment:
so `client` and `contentType` is not needed any more?

Reply:
I don't think they are important enough to justify the additional dimension. Also, I've never seen any open-sourced project adding those to their request related metrics.

Reply:
SGTM